recipes/hiline/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "HiLine" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "d6396ff62c040b96d88ab87b3944328916165c67eb91efca99ef65a0e7248f0e" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - tox-conda.patch
+
+build:
+  number: 0
+  script: "tox -vvv && {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
Why are the tests run as part of the installation? This is very unusual.

Reply:
becuase the testing section doesn't work, it doesn't pull in the requested dependencies https://github.com/bioconda/bioconda-recipes/pull/21006/commits/ddbf2e8cac4d211a8f4da3c62e3bc9e0e2149195

Reply:
I saw now that the Bioconda policy is actually to [not run test suites that are part of the software](https://bioconda.github.io/contributor/guidelines.html#tests), so you should just remove this. [conda-forge has a similar policy](https://conda-forge.readthedocs.io/en/latest/testing.html#should-a-recipe-run-all-of-a-package-s-tests).

Reply:
the test is not part of a test suite and is not part of the software, it is a standalone script that requires pytest to run, i don't want to include it in the repository and i don't want any testing frameworks as runtime dependencies, that's why the testing is done at build time.

Reply:
Which standalone script do you refer to? If it is part of HiLine, then that’s what is meant with "test suites that are part of the software". Don’t run that as part of the recipe at all, neither during build nor add it as a test in the `tests:` section.

Reply:
The implicit assumption behind the policy is that all software gets tested by its authors before it is released.

I see you are the author of the tool for which you’re adding the recipe here. It looks a little bit as if you want to benefit from Bioconda’s CI pipeline so that it will run the tests for you. But the proper way would be for you to set up automated tests in your own repo.